src/scripts.js

```diff
@@ -214,6 +217,9 @@ function showRecipesToCook() {
   let unsavedRecipes = recipes.filter(recipe => {
     return !user.recipesToCook.includes(recipe.id);
   });
+  if(unsavedRecipes.length === recipes.length){
+    return domUpdates.emptyPageErrorMsg('click the mixer on');
+  }
```

Comment:
I like this UX addition, it makes sense to have an error message if they haven't  saved anything